vllm/model_executor/models/ultravox.py

```diff
@@ -346,6 +358,17 @@ def sampler(self):
 
         return get_sampler()
 
+    # Following PR: https://github.com/vllm-project/vllm/pull/7199/files
+    # check language_model and audio_tower prefixes
+    # can't tell if vLLM will apply audio lora or not based on following warning:
+    # https://github.com/vllm-project/vllm/pull/7199/files#diff-d3df23c3e3bcfe97ee8507061c6de54f0eff23a8c75d7f5999062c42245290f8R1033
```

Comment:
QQ:what's mean?

Reply:
sorry comment is not relevant anymore. I'll remove this one